recipes/postgis/build.sh

```diff
@@ -0,0 +1,15 @@
+chmod 755 configure
+./configure \
+    --prefix=$PREFIX \
+    --with-pgconfig=$PREFIX/bin/pg_config \
+    --with-gdalconfig=$PREFIX/bin/gdal-config \
+    --with-xml2config=$PREFIX/bin/xml2-config \
+    --with-projdir=$PREFIX \
+    --with-libiconv=$PREFIX \
+    --with-jsondir=$PREFIX \
+    --with-pcredir=$PREFIX \
+    --with-gettext \
+    --with-raster \
+    --with-topology
+make
+make install
```

Comment:
Is there a `make check` we can run?

Reply:
I am working on getting make check to work. The builds will be in a failure state until that is done.